inst/rmarkdown/templates/quantitative_research_in_psych_prereg/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,411 @@
+---
+title           : "My preregistration based on the Preregistration for Quantitative Research in Psychology Template^[This document was created using the **Preregistration for Quantitative Research in Psychology Template**. The template was developed by a task force composed of members of the American Psychological Association (APA), the British Psychological Society (BPS), the German Psychological Society (DGPs), the Center for Open Science (cos), and the Leibniz Institute for Psychology (ZPID). This work is licensed under the CC BY-NC-SA 4.0 license. To view a copy of the license, visit https://creativecommons.org/licenses/by-nc-sa/4.0/.]"
+shorttitle      : "My preregistration"
+date            : "`r Sys.setlocale('LC_TIME', 'C'); format(Sys.time(), '%d\\\\. %B %Y')`"
+
+author: 
+  - name        : First Author
+    affiliation : 1
+  - name        : Ernst-August Doelle
+    affiliation : "1,2"
+
+affiliation:
+  - id          : 1
+    institution : Wilhelm-Wundt-University
+  - id          : 2
+    institution : Konstanz Business School
+
+header-includes: |
```

Comment:
I'm not sure where to include this (yaml header, normal code, tex file?). I just included this part because I received an error message whenever I knitted to PDF that said that my headheight was too small. Could you specify please?

Reply:
Sorry, I think this just needs to go in `pre_processor()` (line 44 of `quantitative_research_in_psych_prereg.R`) like so:

~~~r
args <- pdf_pre_processor(metadata, input_file, runtime, knit_meta, files_dir, output_dir)
header_includes <- c("\\setlength{\\headheight}{14.0pt}", metadata$`header-includes`)
args <- c(args, "--variable", paste0("header-includes:", paste(header_includes, collapse = "\n")))
args
~~~

The last line is just to return the modified `args` from `pre_processor()`.